pg_wait_sampling.c

```diff
@@ -39,36 +34,42 @@
 
 PG_MODULE_MAGIC;
 
-void		_PG_init(void);
-
+/* Marker whether extension is setup in shared mode */
 static bool shmem_initialized = false;
 
+/* Global settings */
+int MaxProfileEntries = 5000;
+int HistoryBufferSize = 5000;
+int HistoryPeriod = 0;
+int ProfilePeriod = 10;
+bool WhetherProfilePid = true;
+bool WhetherProfileQueryId = true;
+
+/* Function declarations */
+void _PG_init(void);
+// TODO: add void _PG_fini(void);
```

Comment:
_PG_fini was removed from PostgreSQL, so we don't need this TODO item.

Reply:
OK, will be done